libamqpprox/amqpprox_session.cpp

```diff
@@ -798,13 +795,17 @@ void Session::handleSessionError(const char *              action,
 
     if (ec != boost::asio::error::operation_aborted) {
         // TODO notify EventSource
-        LOG_WARN << action << " received error_code=" << ec << " "
-                 << TlsUtil::augmentTlsError(ec) << " conn="
-                 << d_sessionState.hostname(d_sessionState.getIngress().second)
-                 << ":" << d_sessionState.getIngress().second.port() << "->"
-                 << d_sessionState.hostname(d_sessionState.getEgress().second)
-                 << ":" << d_sessionState.getEgress().second.port()
-                 << " direction=" << direction;
+
+        LOG_SEV(ec != boost::asio::ssl::error::stream_truncated
+                    ? Logging::AMQPPROX_LOG_WARN
+                    : Logging::AMQPPROX_LOG_INFO)
+            << action << " received error_code=" << ec << " "
+            << TlsUtil::augmentTlsError(ec) << " conn="
+            << d_sessionState.hostname(d_sessionState.getIngress().second)
+            << ":" << d_sessionState.getIngress().second.port() << "->"
+            << d_sessionState.hostname(d_sessionState.getEgress().second)
+            << ":" << d_sessionState.getEgress().second.port()
+            << " direction=" << direction;
```

Comment:
Something like this might make why we don't mind this error a bit clearer? https://github.com/boostorg/beast/issues/824 suggests to ignore this error when dealing with some tls clients/servers, so if we're seeing this with RabbitMQ maybe we need to drop it to debug.

```suggestion
        if (ec == boost::asio::ssl::error::stream_truncated) {
            LOG_DEBUG << "TLS session truncated on close: conn="
                                 << d_sessionState.connSummary()
                                 << " direction=" << direction;
        } else {
            LOG_WARN << action << "received error_code=" << ec << " " 
                << TlsUtil::augmentTlsError(ec) << " conn="
                << d_sessionState.connSummary()
                << " direction=" << direction;
        }
```

where connSummary looks like:

```cpp
std::stringstream SessionState::connSummary() {
    std::stringstream ss;
    ss << this->hostname(this->getIngress().second)
        << ":" << this->getIngress().second.port << "->"
        << this->hostname(this->getEgress().second)
        << this->getEgress().second.port();
    return ss;
}
```